sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_reschedule_v1/query.sql

```diff
@@ -11,11 +12,12 @@ FROM
 WHERE
   NOT _fivetran_deleted
 UNION ALL
--- including historical data from the pre-migration Airflow instance
+  -- including historical data from the pre-migration Airflow instance
 SELECT
   dag_id,
   task_id,
-  execution_date,
```

Comment:
is this also the same in the historical data? Or did the schema change here? Not sure since I don't have access to the underlying data